test/Swashbuckle.AspNetCore.SwaggerGen.Test/ConfigureSwaggerGeneratorOptionsTests.cs

```diff
@@ -15,4 +21,27 @@ public static void DeepCopy_Copies_All_Properties()
         // to SwaggerGeneratorOptions and ConfigureSwaggerGeneratorOptions.DeepCopy() needs to be updated
         Assert.Equal(18, publicProperties.Length);
     }
+
+    [Fact]
+    public static void AddingDocumentFilterInstance_WhenConfiguringOption_SameInstanceIsAdded()
+    {
+        var webhostingEnvironment = Substitute.For<IWebHostEnvironment>();
+        webhostingEnvironment.ApplicationName.Returns("Swashbuckle.AspNetCore.SwaggerGen.Test");
+
+        var testDocumentFilter = new TestDocumentFilter();
+
+        var options = new SwaggerGenOptions();
+        options.AddDocumentFilterInstance(testDocumentFilter);
+
+        var configureSwaggerGeneratorOptions = new ConfigureSwaggerGeneratorOptions(
+            Options.Create(options),
+            null,
+            webhostingEnvironment);
+        var swaggerGeneratorOptions = new SwaggerGeneratorOptions();
+
+        configureSwaggerGeneratorOptions.Configure(swaggerGeneratorOptions);
+
+        Assert.Single(swaggerGeneratorOptions.DocumentFilters);
```

Comment:
IIRC `Single()` returns that value so this can be:

```csharp
var actual = Assert.Single(swaggerGeneratorOptions.DocumentFilters);
Assert.Same(testDocumentFilter, actual);
```